docs/src/rsh/step/index.md

```diff
@@ -463,7 +463,23 @@ It accepts an optional bytes argument, which is included in any reported violati
 closeTo( Who, after, nonNetPayAmt )
 ```
 
-@{ref("rsh", "closeTo")} Has participant `{!rsh} Anybody` make a publication, then transfer the `{!rsh} balance()` and the non-network pay amount to `{!rsh} Who` and end the DApp after executing the function `{!rsh} after` in a step.
+@{ref("rsh", "closeTo")} Has participant `{!rsh} Anybody` make a publication, then transfer the `{!rsh} balance()` and the non-network pay amount to `{!rsh} Who` and end the dApp after executing the function `{!rsh} after` in a step.
+
+As an example, codifying the prose above would look like:
+
+```reach
+// Has participant `{!rsh} Anybody` make a publication
```

Comment:
Does this render properly? I assume it doesn't

Reply:
Ticks don't render in comments. We can use single quotes to indicate signification in comments. e.g.
// Has participant 'Anybody' make a publication
// Transfer the balance() and the non-network payment amount to 'Who'
// End the DApp after executing the function 'after' in a step

Reply:
Sounds good